src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -224,7 +237,7 @@ private async Task<PublicationTreeTopicViewModel> BuildPublicationTreeTopic(Topi
             .Where(publication => publication.LatestPublishedReleaseId != null)
             .ToAsyncEnumerable()
             .SelectAwait(async publication =>
-                await BuildPublicationTreePublication(publication))
+                await BuildPublicationTreePublication(publication)) 
```

Comment:
Minor, but no overall change here, just some whitespace creeping in on L240 and L279 as well.

Reply:
I suspect you've made these changes and they've got lost somewhere? I'll unresolve any comments where I think the change hasn't been made.